jobspec/test-fixtures/basic.hcl

```diff
@@ -22,6 +22,7 @@ job "binstore-storagelocker" {
     health_check = "manual"
     min_healthy_time = "10s"
     healthy_deadline = "10m"
+    progress_deadline = "10m"
```

Comment:
Placeholder comment- if these are new user options, make sure they are correctly documented.

Reply:
Will do documentation in a different PR